go.sum

```diff
@@ -261,6 +261,8 @@ github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e h1:fD57ERR4JtEqsWb
 github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e/go.mod h1:zD1mROLANZcx1PVRCS0qkT7pwLkGfwJo4zjcN/Tysno=
 github.com/nishanths/exhaustive v0.0.0-20200525081945-8e46705b6132 h1:NjznefjSrral0MiR4KlB41io/d3OklvhcgQUdfZTqJE=
 github.com/nishanths/exhaustive v0.0.0-20200525081945-8e46705b6132/go.mod h1:wBEpHwM2OdmeNpdCvRPUlkEbBuaFmcK4Wv8Q7FuGW3c=
```

Comment:
This is go.mod issue, which didn't clean up the old version.

If you want you can re-create go.sum files, so that old entries are removed.

Reply:
Did you mean edit `go.sum` by hand? I went ahead and added a commit that does that.

Reply:
Update: I see that `go mod tidy` does what's needed. I ran that instead.

Reply:
I meant that we can remove go.sum and run `go mod tidy` to recreate it again :).

Reply:
Awesome, thanks. 

Running `rm go.sum && go mod tidy` didn't produce any more changes.

Reply:
this is one of the thing that I don't like with go mod :(